osu.Game.Rulesets.Osu/Mods/OsuModHoldOff.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using osu.Framework.Bindables;
+using osu.Framework.Localisation;
+using osu.Framework.Utils;
+using osu.Game.Beatmaps;
+using osu.Game.Configuration;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Osu.Beatmaps;
+using osu.Game.Rulesets.Osu.Objects;
+using osu.Game.Rulesets.Osu.Utils;
+
+namespace osu.Game.Rulesets.Osu.Mods
+{
+    public class OsuModHoldOff : Mod, IApplicableAfterBeatmapConversion
+    {
+        public override string Name => "Hold Off";
+
+        public override string Acronym => "HO";
+
+        public override double ScoreMultiplier => 1;
+
+        public override LocalisableString Description => @"Convert all sliders to streams.";
+
+        public override ModType Type => ModType.Conversion;
+
+        public override Type[] IncompatibleMods => new[] { typeof(OsuModTarget), typeof(OsuModStrictTracking) };
+
+        [SettingSource("Beat divisor")]
+        public BindableInt BeatDivisor { get; } = new BindableInt(4)
+        {
+            MinValue = 1,
+            MaxValue = 16
+        };
+
+        [SettingSource("Maximum stream length", "Streams longer than this value will be lightened.")]
```

Comment:
you should probably add something like " length in beat measures" or "duration in beats " because most people wont know what you mean, or assume the sliders are being capped in actual on screen length